src/Elasticsearch.Net/Utf8Json/Formatters/ValueTupleFormatter.cs

```diff
@@ -174,7 +172,7 @@ static ValueTupleFormatterHelper()
 		}
 	}
 
-	#if DOTNETCORE
+	#if NETSTANDARD
```

Comment:
Should this be `NETSTANDARD2_1`?

Reply:
`ValueTuple<>` exists in `netstandard2.0` 
`ValueTask<>` only exists in `netstandard2.1`